app/config/config_test.yml

```diff
@@ -125,3 +125,11 @@ doctrine:
             default:
                 logging: false
                 profiling: false
+
+api_platform:
+    enable_docs: false
```

Comment:
not needed as you do it in the global config file

Reply:
`config_test.yml` extends `config_dev.yml` that changes these values.